source/howtos/updating_pelican_sites.rst

```diff
@@ -0,0 +1,601 @@
+Updating CASS Pelican Websites
+==============================
+
+This documentation applies to both the CASS main website, the OSL website and 
+any other sites using the Pelican static site generator.
```

Comment:
I would consider linking to a webpage explaining what SSG's are here. If this is new information for the reader it's good to put it in a hyperlink.